src/adapters/methods/bottleneck.py

```diff
@@ -94,28 +94,6 @@ def add_adapter(self, adapter_name: str, layer_idx: int) -> bool:
 
         return False
 
-    def average_adapter(self, adapter_name: str, input_adapters: Dict[str, float]) -> bool:
```

Comment:
Where did this method go?

Reply:
Bottleneck adapters simply use the `average_adapter` method of `AdapterLayerBase`